src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -59,18 +69,27 @@ public void launchTask(final ExecutorDriver driver, final TaskInfo taskInfo) {
           .setTaskId(journalNodeTask.taskInfo.getTaskId())
           .setState(TaskState.TASK_RUNNING)
           .build());
+      healthCheckJN = new TimedHealthCheck(driver, journalNodeTask);
+      timer.scheduleAtFixedRate(healthCheckJN, schedulerConf.getHealthCheckWaitingPeriod(),
+          schedulerConf.getHealthCheckFrequency());
```

Comment:
my apologies, @adam-mesos. I thought with one ship it review, this would be ok. I will gladly revert. Mesos health checks are fine, but I thought it was cleaner here to write the java code directly as opposed to distributing a binary and running it as a shell command. We can discuss before I submit this PR.